build.sbt

```diff
@@ -539,6 +549,30 @@ lazy val compilerBridge211 = compilerBridge.jvm(scala211)
 lazy val compilerBridge212 = compilerBridge.jvm(scala212)
 lazy val compilerBridge213 = compilerBridge.jvm(scala213)
 
+lazy val compilerBridge213_next = (project in (file(".sbt") / "matrix" / "compilerBridge2_13_next"))
+  .dependsOn(compilerInterface.jvm(false))
+  .settings(
+    scalaVersion := scala213_next,
+    autoScalaLibrary := false,
+    baseSettings,
+    compilerVersionDependentScalacOptions,
+    // We need this for import Compat._
+    Compile / scalacOptions --= Seq("-Ywarn-unused-import", "-Xfatal-warnings"),
+    libraryDependencies += scalaCompiler.value % "provided",
+    exportJars := true,
+    Compile / unmanagedResourceDirectories ++= (compilerBridge213 / Compile / unmanagedResourceDirectories).value,
+    Compile / unmanagedSourceDirectories ++= (compilerBridge213 / Compile / unmanagedSourceDirectories).value,
+    Compile / sources := {
+      val xs = (Compile / sources).value
+      (xs.filterNot { x => x.getName() == "DelegatingReporter.scala" }) ++
+        Seq(
+          (compilerBridge213 / Compile / scalaSource).value.getParentFile() / "resources" / "scala-2.13.12" / "DelegatingReporter.scala"
```

Comment:
What is the reason to put this file in `resources`, vs the more common approach to have a directory in `src`? There are already version folders in internal/compiler-bridge/src/main.

Reply:
Currently compiler bridge serves all of 2.13.x series using https://repo1.maven.org/maven2/org/scala-sbt/compiler-bridge_2.13/1.9.2/, so the `src/main` would be have the default implementation that works with 2.13.11, and I was gonna implement the source swapping in ZincComponentCompiler - https://github.com/sbt/sbt/blob/1.9.x/zinc-lm-integration/src/main/scala/sbt/internal/inc/ZincComponentCompiler.scala

Reply:
Ah, so the `compilerBridge213_next` project here is just for testing, not for releasing a different artifact.